<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nWinList</class>
 <widget class="QMainWindow" name="nWinList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinList</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="nTreeWidget" name="images">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>45</number>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Short</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Origin</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Scale</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hidden</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="pans">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCopy"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionShort"/>
   <addaction name="actionName"/>
   <addaction name="actionOrigin"/>
   <addaction name="actionScale"/>
   <addaction name="actionFreeze"/>
   <addaction name="actionPans"/>
  </widget>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveClipboard.png</normaloff>:icons//saveClipboard.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="toolTip">
    <string>Duplicate image(s)</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset>
     <normaloff>:icons//remove.png</normaloff>:icons//remove.png</iconset>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
   <property name="toolTip">
    <string>Remove image(s)</string>
   </property>
  </action>
  <action name="actionShort">
   <property name="icon">
    <iconset>
     <normaloff>:icons//short.png</normaloff>:icons//short.png</iconset>
   </property>
   <property name="text">
    <string>Short</string>
   </property>
   <property name="toolTip">
    <string>Change shortname</string>
   </property>
  </action>
  <action name="actionName">
   <property name="icon">
    <iconset>
     <normaloff>:icons//name.png</normaloff>:icons//name.png</iconset>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="toolTip">
    <string>Change name</string>
   </property>
  </action>
  <action name="actionOrigin">
   <property name="icon">
    <iconset>
     <normaloff>:icons//center.png</normaloff>:icons//center.png</iconset>
   </property>
   <property name="text">
    <string>Origin</string>
   </property>
   <property name="toolTip">
    <string>Change origin</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="icon">
    <iconset>
     <normaloff>:icons//scale.png</normaloff>:icons//scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="toolTip">
    <string>Change scale</string>
   </property>
  </action>
  <action name="actionPans">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:icons//winlist.png</normaloff>:icons//winlist.png</iconset>
   </property>
   <property name="text">
    <string>Pans</string>
   </property>
  </action>
  <action name="actionFreeze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/lockOpen.png</normaloff>
     <normalon>:/icons/lockClose.png</normalon>:/icons/lockOpen.png</iconset>
   </property>
   <property name="text">
    <string>freeze</string>
   </property>
   <property name="toolTip">
    <string>Set current origin/scale to new images</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">../src/pans/winlist/nTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPans</sender>
   <signal>triggered(bool)</signal>
   <receiver>pans</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPans</sender>
   <signal>toggled(bool)</signal>
   <receiver>images</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
